protocol-tokio/src/protocol/codec/handshake.rs

```diff
@@ -17,7 +17,7 @@ use super::{MessageCode, MessageType};
 pub struct ProtocolMagic(u32);
 impl Default for ProtocolMagic {
     fn default() -> Self {
-        ProtocolMagic(764824073)
+        ProtocolMagic(1097911063)
```

Comment:
add a comment that this should be replaced by a configuration/setting mechanism...


```rust
// TODO: get this from user input
```

and for now I prefer the default to be mainnet's protocol magic.

Reply:
I think it would be less error-prone to define the magic in the code that implements the protocol, rather than allow it to be redefined with an arbitrary value.

Reply:
Upon reading the discussion above, I agree that it should be obtained from the same source that defines the genesis block hash.